docs/examples/frameworks/jax/pax-basic_example.ipynb

```diff
@@ -0,0 +1,1482 @@
+{
```

Comment:
<p><strong>The f</strong><s>F</s>irst step (...)</p><p> </p><p>For more information on DALI pipeline<strong>s</strong> look (...)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/5060/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Maybe "on writing DALI pipelines"? </p>

Reply:
<p>Done</p>